tests/README.md

```diff
@@ -1,24 +1,130 @@
 # Integration tests
 
-Integration tests require a running Kubevirt cluster.  Once you have a running
-Kubevirt cluster, you can use the `-master` and the `-kubeconfig` flags to
-point the tests to the cluster.
+Kubevirt integration tests are a suite of end-to-end tests which run
+against an active Kubevirt cluster.
 
-## Run them on an arbitrary KubeVirt installation
+The tests may execute against various clusters, from integrated kubevirtci
+deployments and up to external kubernetes or OpenShift ones.
+
+## Kubevirtci
+
+[Kubevirtci](https://github.com/kubevirt/kubevirtci/)
+provides containerized Kubernetes clusters which can be used
+to test KubeVirt and similar projects.
+
+KubeVirt integration tests are being run against kubevirtci clusters on CI
+and is the simplest way to run the tests locally in a development environment.
 
+To run the test locally:
+- Configure the desired cluster setup (provider, number of nodes and nics).
+
+   > **Note:** Some tests may run only on specific configuration and hardware.
+
+   > **Note:** For detailed instructions on how to work with a specific provider,
+        please explore kubevirtci providers, e.g.
+  [k8s-1.18](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/k8s-1.18)
+  
+  The following configuration is of a k8s-1.18 provider with two nodes, each
+  having two additional interfaces (3 in total).
 ```
-cd tests # from the git repo root folder
-go test -kubeconfig=path/to/my/config -config=default-config.json
+export KUBEVIRT_PROVIDER=k8s-1.18
+export KUBEVIRT_NUM_NODES=2
+export KUBEVIRT_NUM_SECONDARY_NICS=2
 ```
 
-## Run them on one of the core KubeVirt providers
+- Raise a cluster based on the configuration.
+  At this stage, it will not contain kubevirt.
```

Comment:
nice, but do we want duplication of it ? 
we have this section on other docs

Reply:
If I am working on kubevirt/tests, I want to read how to run them.
I do not think there is any other documentation presenting this under kubevirt, but if there is, please share.

Reply:
docs/getting-started.md

Reply:
Ok, thanks.
I have just improved the instructions that appeared here in the first place and I think these still have a place here because they are close to the actual e2e tests.

While this document focuses on tests, that document is about everything.
The procedures here are e2e tests oriented and there an attempt to generalize it between running a regular cluster from sources and developing instructions.

Bottom line, it will be odd to ref the getting-started doc here and if any ref is needed is the reverse.